src/NUnitTestAdapter/NUnit.TestAdapter.csproj

```diff
@@ -32,19 +32,12 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Testing.Extensions.VSTestBridge" Version="1.0.1" />
```

Comment:
The bridge as a deps on object model (Microsoft.TestPlatform.ObjectModel >= 17.5.0). If that's a too big constraint for you, we need to do some manual tests here to see if we would break something when doing a forced downgrade here.